pkg/scheduler/api/node_info.go

```diff
@@ -46,8 +46,7 @@ type NodeInfo struct {
 
 	Tasks map[TaskID]*TaskInfo
 
-	// Used to store custom information
-	Others map[string]interface{}
+	GPUDevices map[int]*GPUDevice
```

Comment:
please do not remove `Others`, some projects are using it.

Reply:
reverted